gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/v4/DefaultApiReactor.java

```diff
@@ -344,25 +422,43 @@ protected void doStart() throws Exception {
         long startTime = System.currentTimeMillis(); // Get the start Time
 
         // Start resources before
+        resourceLifecycleManager.start();
         policyManager.start();
 
         // Create securityChain once policy manager has been started.
         this.securityChain = new SecurityChain(api.getDefinition(), policyManager, ExecutionPhase.REQUEST);
 
+        if (tracingEnabled) {
+            processorChainHooks.add(new TracingHook("processor-chain"));
+            invokerHooks.add(new TracingHook("invoker"));
+            securityChain.addHooks(new TracingHook("security-plan"));
+        }
+
+        api
+            .getDefinition()
+            .getListeners()
+            .stream()
+            .filter(listener -> listener.getType() == ListenerType.HTTP)
```

Comment:
only for http listener? What about subscription?

Reply:
I just wanted to include the logging to support sync case. Logging is a dedicated task. Things will certainly evolve in the future.